ratpack-session/src/test/groovy/ratpack/session/clientside/ClientSideSessionSpec.groovy

```diff
@@ -48,7 +47,8 @@ class ClientSideSessionSpec extends SessionSpec {
     "DESede/CBC/NoPadding",
     "DESede/CBC/PKCS5Padding",
     "DESede/ECB/NoPadding",
-    "DESede/ECB/PKCS5Padding"
+    "DESede/ECB/PKCS5Padding",
+    "DESede/CBC/NoPadding"
```

Comment:
So, this algorithm is already in the list above. Line 47.

Reply:
Ahh, good find. I thought since we were testing it separately in the annotated test that we weren't also testing it above.